src/test/kotlin/org/elm/ide/inspections/TypeInferenceInspectionTest.kt

```diff
@@ -370,7 +370,7 @@ main : Maybe People
 main = person George
 --^
 
---@ Foo.elm
+--@ People/Washington.elm
```

Comment:
This was a bug in the original test, but it was being masked by my loosey-goosey fallback when the ElmProject was unknown (which it was in basically all of the tests, including this one).